helper/ios_test.go

```diff
@@ -3,20 +3,22 @@ package helper
 import (
 	"testing"
 
-	"github.com/ProtonMail/gopenpgp/constants"
-	"github.com/ProtonMail/gopenpgp/crypto"
+	"github.com/ProtonMail/gopenpgp/v2/constants"
+	"github.com/ProtonMail/gopenpgp/v2/crypto"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestIOSSignedMessageDecryption(t *testing.T) {
-	testPrivateKeyRing, _ := crypto.BuildKeyRingArmored(readTestFile("keyring_privateKey", false))
-	testPublicKeyRing, _ := crypto.BuildKeyRingArmored(readTestFile("mime_publicKey", false))
-
+	privateKey, _ := crypto.NewKeyFromArmored(readTestFile("keyring_privateKey", false))
```

Comment:
We should avoid ignoring errors, even in test functions.

Reply:
OK, added to plan for 2.0.1 under test improvements